### DISCUSSION TITLE
Suggestion: GitHub stars have to be earned

### DISCUSSION THREAD

**Discussion by xpe** [terminusdb/terminusdb#2118] [General]
Some direct feedback for you. I just read the following from your README, and it puts me off:

> If you are reading this, give this repo a star.

Please remove this; it smacks of desperation and begging.\*

People tend to star a project if/when they are ready.

If you feel like you _must_ ask, I would suggest:

- [avoid the command form because you are asking for a favor](https://english.stackexchange.com/a/211032/28940)
- don't put it at the very top; you are wasting your precious opportunity to make a great first impression
- be humble ("if you find this useful, please star this repo")

\* Purely along your project's self-interest, I would expect this language hurts your project more than it helps.

**Comment by alexander-mart** [comment]
@xpe I propose a PR: #2159

**Reply by alexander-mart** [reply]
@xpe @hoijnet PR is merged and now this topic can be closed
